tools/tertiary-analysis/scanpy/anndata_operations.xml

```diff
@@ -158,6 +215,33 @@ adata.write('output.h5', compression='gzip')
       <param name="flag" type="text" label="Var name" help="Name of the column in var.names where this boolean flag is stored, for example 'mito' for mitochondrial genes."/>
     </repeat>
     <param name="top_genes" label="Number of top genes" value='50' help="to calculate percentage of the flagged genes in that number of top genes. Used by sc.pp.calculate_qc_metrics (integer)." type="integer"/>
+    <conditional name="copy_r">
+      <param name="default" type="boolean" checked="false" label="Copy adata.X to adata.raw"/>
+      <when value="true">
+        <param name="r_source" type="data" label="AnnData object with to copy to .raw" help="Copies adata (subset to matching obs) from this AnnData object into the main input as .raw. Make sure to use an AnnData object containing all .obs in the main input." format="h5,h5ad" />
```

Comment:
Label is unclear.

Reply:
In the help: Subset to matching obs or Subject to matching obs?

Reply:
Thanks- label fixed. 'subset' is correct, I've set it to automatically subset the input raw anndata to the obs of the main object, provided all of the latter are present in the former.